### PR TITLE
[Admin] (miscellaneous) adding descriptions to articles

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1a76f8f](https://github.com/OfficeDev/office-js-docs-pr/commits/1a76f8f6977aa104297b3488a28ac78753e5c427)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/resource-limits-and-performance-optimization.md) | :warning:Warning | | [Details](#user-content-29829ab4-97030dcd)
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/design/accessibility-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/accessibility-guidelines.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-color.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-color.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-design-language.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-design-language.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-design.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-design.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons-fresh.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons-fresh.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons-monoline.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons-monoline.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-layout.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-layout.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-typography.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-typography.md) | :white_check_mark:Succeeded | |
[docs/design/authentication-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/authentication-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/branding-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/branding-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/data-visualization-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/data-visualization-guidelines.md) | :white_check_mark:Succeeded | |
[docs/design/design-toolkits.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/design-toolkits.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/design/first-run-experience-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/first-run-experience-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/interface-elements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/interface-elements.md) | :white_check_mark:Succeeded | |
[docs/design/navigation-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/navigation-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/office-ui-fabric.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/office-ui-fabric.md) | :white_check_mark:Succeeded | |
[docs/design/using-motion-office-addins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/using-motion-office-addins.md) | :white_check_mark:Succeeded | |
[docs/design/ux-design-pattern-templates.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/ux-design-pattern-templates.md) | :white_check_mark:Succeeded | |
[docs/design/voice-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/voice-guidelines.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/add-ins-with-angular2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/add-ins-with-angular2.md) | :white_check_mark:Succeeded | |
[docs/develop/addressing-same-origin-policy-limitations.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/addressing-same-origin-policy-limitations.md) | :white_check_mark:Succeeded | |
[docs/develop/asynchronous-programming-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/asynchronous-programming-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/auth-with-office-dialog-api.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/auth-with-office-dialog-api.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/get-javascript-intellisense-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/get-javascript-intellisense-in-visual-studio.md) | :white_check_mark:Succeeded | |

<a id="29829ab4-97030dcd"></a>
### [docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/resource-limits-and-performance-optimization.md)
 - **[Warning]** `````(Line: 2, Col: 54, Idx: 124) - (Line: 2, Col: 99, Idx: 169): While parsing a block mapping, did not find expected key.`````
 - **[Warning]** `````Missing attribute: title. Add a title string to show in search engine results.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162120277034-1668%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AqB%2BiFY3vYxXYga0I9M4roc8B24HgLJWlNud0A3lqVU%3D&st=2020-03-16T21%3A16%3A43Z&se=2020-04-16T21%3A21%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/design/add-in-icons-monoline.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Monoline style icon guidelines for Office Add-ins
-description: ''
+description: 'Get guidelines for using Monoline style icon icons in Office Add-ins.'
```
Comment (lindalu-MSFT) [inline docs/design/add-in-icons-monoline.md]:
```suggestion
description: 'Get guidelines for using Monoline style icons in Office Add-ins.'
```
"icon icons" just a bit too too.

File: docs/design/add-in-icons-fresh.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Fresh style icon guidelines for Office Add-ins
-description: ''
+description: 'Get guidelines for using Fresh style icon icons in Office Add-ins.'
```
Comment (lindalu-MSFT) [inline docs/design/add-in-icons-fresh.md]:
```suggestion
description: 'Get guidelines for using Fresh style icons in Office Add-ins.'
```

File: docs/design/add-in-color.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Color guidelines for Office Add-ins
-description: ''
+description: 'Learn how to use colors in the UI of an Office Add-in.'
```
Comment (lindalu-MSFT) [inline docs/design/add-in-color.md]:
```suggestion
description: 'Learn how to use colors in the user interface (UI) of an Office Add-in.'
```

File: docs/design/add-in-typography.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Typography guidelines for Office Add-ins
-description: ''
+description: 'Learn what typefaces and font sizes to use in Office Add-ins.'
```
Comment (lindalu-MSFT) [inline docs/design/add-in-typography.md]:
```suggestion
description: 'Learn about typography guidelines such as which typefaces and font sizes to use in Office Add-ins.'
```

File: docs/design/dialog-boxes.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Dialog boxes in Office Add-ins
-description: ''
+description: 'Learn best practices for the visual design of dialogs in Office Add-ins.'
```
Comment (lindalu-MSFT) [inline docs/design/dialog-boxes.md]:
```suggestion
description: 'Learn best practices for the visual design of dialog boxes in Office Add-ins.'
```

File: docs/design/design-toolkits.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Design toolkits for Office Add-ins
-description: ''
+description: 'Get links to some tools to help design the UI of an Office Add-in.'
```
Comment (lindalu-MSFT) [inline docs/design/design-toolkits.md]:
```suggestion
description: 'Learn about tools to help you design the user interface (UI) of an Office Add-in.'
```

File: docs/design/interface-elements.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Office UI elements for Office Add-ins
-description: ''
+description: 'Get an overview of the different kinds of UI elements in an Office Add-in.'
```
Comment (lindalu-MSFT) [inline docs/design/interface-elements.md]:
```suggestion
description: 'Get an overview of the different kinds of user interface (UI) elements in an Office Add-in.'
```

File: docs/design/ux-design-pattern-templates.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: UX design patterns for Office Add-ins
-description: ''
+description: 'Get an overview of the UI design patterns for Office Add-ins, including patterns for navigation, authentication, first-run, and branding.'
```
Comment (lindalu-MSFT) [inline docs/design/ux-design-pattern-templates.md]:
```suggestion
description: 'Get an overview of the user interface (UI) design patterns for Office Add-ins, including patterns for navigation, authentication, first-run, and branding.'
```

File: docs/develop/asynchronous-programming-in-office-add-ins.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Asynchronous programming in Office Add-ins
-description: ''
+description: 'Learn how the Office JavaScript library uses asynchronous programming in Office Add-ins.'
```
Comment (lindalu-MSFT) [inline docs/develop/asynchronous-programming-in-office-add-ins.md]:
```suggestion
description: 'Learn how the Office JavaScript API library uses asynchronous programming in Office Add-ins.'
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2eb0484](https://github.com/OfficeDev/office-js-docs-pr/commits/2eb0484c428f36690be3ba169e70e0ab46e1bf82)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/concepts/requirements-for-running-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/requirements-for-running-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/concepts/resource-limits-and-performance-optimization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/concepts/resource-limits-and-performance-optimization.md) | :white_check_mark:Succeeded | |
[docs/design/accessibility-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/accessibility-guidelines.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-color.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-color.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-design-language.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-design-language.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-design.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-design.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons-fresh.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons-fresh.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons-monoline.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons-monoline.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-icons.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-icons.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-layout.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-layout.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-typography.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/add-in-typography.md) | :white_check_mark:Succeeded | |
[docs/design/authentication-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/authentication-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/branding-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/branding-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/data-visualization-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/data-visualization-guidelines.md) | :white_check_mark:Succeeded | |
[docs/design/design-toolkits.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/design-toolkits.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/design/first-run-experience-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/first-run-experience-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/interface-elements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/interface-elements.md) | :white_check_mark:Succeeded | |
[docs/design/navigation-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/navigation-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/office-ui-fabric.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/office-ui-fabric.md) | :white_check_mark:Succeeded | |
[docs/design/using-motion-office-addins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/using-motion-office-addins.md) | :white_check_mark:Succeeded | |
[docs/design/ux-design-pattern-templates.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/ux-design-pattern-templates.md) | :white_check_mark:Succeeded | |
[docs/design/voice-guidelines.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/design/voice-guidelines.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/add-ins-with-angular2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/add-ins-with-angular2.md) | :white_check_mark:Succeeded | |
[docs/develop/addressing-same-origin-policy-limitations.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/addressing-same-origin-policy-limitations.md) | :white_check_mark:Succeeded | |
[docs/develop/asynchronous-programming-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/asynchronous-programming-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/auth-with-office-dialog-api.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/auth-with-office-dialog-api.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/get-javascript-intellisense-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/rickki-descriptions-branch/docs/develop/get-javascript-intellisense-in-visual-studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C16%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202003162154078864-1668%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Doe36AUN%2BVzPM2DXG8n80ezYzzTt8cVjXrjIveY87pE%3D&st=2020-03-16T21%3A50%3A38Z&se=2020-04-16T21%3A55%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**